mautic/mautic
# Fix API creating tag duplicates

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features"
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
Creating new tags through API with special characters (e.q `"`, `'`) or whitespace in the beginning/end of the string causes duplicates.

Relates to #6057 #7741

#### Steps to test this PR:
1. Apply this PR
2. Create new tag with API `POST /api/tags/new` with data:
```
{
    "tag": "hello'world"
}
```
3. Note the ID and repeat previous request with same data
4. Only one tag should be created, second call should return the same tag ID
